scripts/arm32_ci_script.sh

```diff
@@ -212,7 +212,7 @@ do
         ;;
     --softfp)
```

Comment:
I agree with you. 
It seems that `-softfp` option is passed by CI. To modify this name, we have to modify CI trigger command as well.